tv_apps/smart-system/js/action_menu.js

```diff
@@ -204,6 +195,34 @@
           this.oncancel();
           break;
       }
+    },
+
+    /**
+     * Whether or not the ActionMenu is visible.
+     * @memberof ActionMenu.prototype
+     * @return {Boolean} if the container is invisible, return false
+     * @return {Number} if the container is visible, return z-index
+     */
+    isVisible: function() {
```

Comment:
Please split this method as two: isVisible return boolean and getOrder returns number.